### PR TITLE
Unittest strategies

### PR DESCRIPTION
**Pull request by pprados**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
I propose a Gekko mock object to test a strategy.
```
const config = require('../../core/util.js').getConfig();
config.MACD={ ... }
const mock=MockGekko(require('../../strategies/MACD.js'));
const tradeHistory=mock.inject(candles).getTradeHistory();
assert(...
```
>node test/strategies/MACD.js --config test/test-config.json

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
add test/strategies/mockGekko.js

* **Other information**:

### CODE REVIEW

**Comment by askmike** [comment]
Hey! As discussed in #1078 I can't merge the labgekko in since I want to keep this out of the core library. Can you update this PR so it does not include those changes (easiest would be to create a new branch for testing strategies).

**Comment by pprados** [comment]
I remove the labgekko in my branch unittest.

**Comment by oktapodia** [comment]
You must create the PR on the `develop` branch, see [contributing](https://github.com/askmike/gekko/blob/stable/CONTRIBUTING.md#new-feature-and-code-contributions)

**Comment by pprados** [comment]
All the link are dead ! Impossible to read the documentation.

**Comment by pprados** [comment]
I rebase the modification to develop branch.

File: test/strategies/TSI.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    persistence: 1
+  }
+};
+const MockGekko=require("./mockGekko.js");
+
+
+const prices = [81, 24, 75, 21, 34, 25, 72, 92, 99, 2, 86, 80, 76, 8, 87, 75, 32, 65, 41, 9, 13, 26, 56, 28, 65, 58, 17, 90, 87, 86, 99, 3, 70, 1, 27, 9, 92, 68, 9];
+
+const trades = [{"price":81,"amount":1,"date":"2015-01-01 00:00:00","profit":81},
+	{"price":24,"amount":-1,"date":"2015-01-01 01:00:00","profit":0},
```
Comment (daviddahl) [inline test/strategies/TSI.js:20]: Question: This data is the same in each of these test strategies file - would it make sense to make it into its own module?

**Pull request closed without merging**